index.md

```diff
@@ -34,32 +37,42 @@ Zowe consists of the following main components.
 
 <section class="bluebackground">
 
-<h1 id="download">DOWNLOAD</h1>
+<h1 id="documentation">Try Zowe now</h1>
+<p>
+Try Zowe today at no charge, and with no installation required. Your no-charge trial is available in two hours for three days. Register and get started today. 
+</p>
+<button><a href="https://www.ibm.com/account/reg/signup?formid=urx-38870">Request a trial</a></button>
+
```

Comment:
Can we add some verbiage under the link like...

"By Proceeding to the trial, you will be leaving the zowe.org website."

Also, can you change the link to https://www.openmainframeproject.org/projects/zowe/ztrial ( which will forward to the same URL ).

Reply:
The requested updates are made. : )